inference-engine/thirdparty/clDNN/kernel_selector/core/kernel_selector.cpp

```diff
@@ -25,6 +26,7 @@
 #endif  // ENABLE_ENV_PRINT
 
 #define ENABLE_OFFLINE_TUNING_CACHE 1
+#define MIN_VERBOSE_LEVEL 3
```

Comment:
MIN might be misread. why not just use 3? I think it is a redundant macro here, while other source code does not use such macro.

Reply:
Removed it. thanks.